hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+
+.TeamCard {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    h2 {
+        margin-bottom: 10px;
+    }
+    .paper {
+        flex-grow: 1;
+        padding: 8px 0px;
+        overflow-x: scroll;
+        overflow-y: scroll;
+        .name {
+            display: flex;
+            align-items: center;
+            &:hover {
+                background-color: color(light2);
+            }
```

Comment:
remove all this since we don't need a hover functionality. 

Add `padding-top: 10px; padding-bottom: 10px;`